fairlearn/reductions/_exponentiated_gradient/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from .exponentiated_gradient import ExponentiatedGradient  # noqa: F401
+from .exponentiated_gradient import ExponentiatedGradientResult  # noqa: F401
+
+__all__ = [
+    "ExponentiatedGradieht",
```

Comment:
typo

Reply:
Hmmm... I wonder how this was working then :-/